jupyterhub/tests/conftest.py

```diff
@@ -43,73 +46,85 @@ def db():
 
 @fixture
 def io_loop():
-    """Get the current IOLoop"""
+    """Create a Tornado IOLoop and make it current."""
     loop = ioloop.IOLoop()
     loop.make_current()
     return loop
 
 
 @fixture(scope='module')
 def app(request):
+    """Start a Hub"""
+    # TODO - Q: what tracks the MockHub instance so it can be cleared later.
```

Comment:
@minrk Thanks! If it's cool with you, I will likely flag questions like this on WIP PRs when I have specific code questions.

Also, you might find this article by Miguel Grinberg interesting: https://blog.miguelgrinberg.com/post/unit-testing-asyncio-code I haven't dug into it yet but it likely contains some good nuggets.